shared/chat/conversation/list.desktop.js

```diff
@@ -142,6 +144,27 @@ class ConversationList extends Component<void, Props, State> {
     this._onScrollSettled()
   }, 100)
 
+  showPopup (message: Message, event: any) {
+    if (message.type !== 'Text') return
+    const clientRect = event.target.getBoundingClientRect()
```

Comment:
Yeah I did that first, but then I needed the popup to dismiss if you click outside (like another conversation or app tab).

I couldn't see an easy way to capture that, and thought dealing with a window level component was easy and worked well and didn't have any weird side effects and also stayed out of the react-virtualized DOM tree which I'm (unreasonably) scared about.

But I could be missing something?